make_deb

```diff
@@ -116,6 +116,13 @@ libdir=${FOGLAMP_ROOT}/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRA
 libdir=${FOGLAMP_ROOT}/cmake_build/C/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
 [ "$libPathSet" -eq "0" ] && echo "Unable to set/update LD_LIBRARY_PATH to include path of Foglamp shared libraries: check whether ${FOGLAMP_ROOT}/lib or ${FOGLAMP_ROOT}/cmake_build/C/lib exists" && exit 1
 
+if [ "$(dpkg -s libcurl4-openssl-dev | grep Status)" == "Status: install ok installed" ]; then
+  echo "libcurl4-openssl-dev library is installed"
+else
+  echo "libcurl4-openssl-dev library is not present on the machine, installing before building the package"
+  ./requirements.sh
+fi
```

Comment:
for future we should keep this if else check in `requirements.sh` itself and if already installed we should print `apt-cache show package-name | grep Version`